airbyte-cdk/python/airbyte_cdk/sources/streams/http/exceptions.py

```diff
@@ -31,6 +34,9 @@ def __init__(self, backoff: Union[int, float], request: requests.PreparedRequest
         :param request: the request that triggered this backoff exception
         :param response: the response that triggered the backoff exception
         """
+        logger.info(
+            f"Backing off for {backoff}s. Request URL: {request.url}, Response Code: {response.status_code}, Response Text: {response.text}"
+        )
```

Comment:
This is a fun one because we don't know how long the `DefaultBackoffException` will backoff for.

Are we not getting the logs from https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/http/rate_limiting.py#L32 and https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/http/rate_limiting.py#L61??